sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1601,7 +1601,7 @@ private void showDisabledToast() {
     synchronized void storeDownloadRequestId(DownloadManager downloadManager, DownloadTask task, long downloadId, long enqueueTime) {
 
         /* Check for if state changed and task not canceled in time. */
-        if (mDownloadTask == task) {
+        if (mDownloadTask == task && mReleaseDetails != null) {
```

Comment:
Should the `else` that goes with this `if` be changed to an `else if`? I guess before it would execute regardless of whether `mReleaseDetails == null`